docs/scripting/functions/CallLocalFunction.md

```diff
@@ -43,4 +43,4 @@ CallLocalFunction crashes the server if it's passing an empty string.
 
 ## Related Functions
 
-- CallRemoteFunction: Call a function in any loaded script.
+- [CallRemoteFunction](../../scripting/functions/CallRemoteFunction.md): Call a function in any loaded script.
```

Comment:
You're already in the `scripting` directory so no need to go up two directories, just `../functions` is necessary

Reply:
Unfortunately I've seen your reply after doing hyperlinks for I and K so they are done the same way.
Thanks for you suggestion, I'll do the rest that way.